lib/reporters/cli/cli-utils.js

```diff
@@ -93,14 +93,13 @@ cliUtils = {
      * color:     |  noTTY:
      * 'on'      -> false
      * 'off'     -> true
-     * otherwise -> Based on CI or isTTY.
+     * otherwise -> Based on isatty.
      *
      * @param {String} color - A flag to indicate usage of the --color option.
      * @returns {Boolean} - A boolean value depicting the result of the noTTY check.
      */
     noTTY: function (color) {
-        return (color === 'off') || (color !== 'on') &&
-            (Boolean(process.env.CI) || !process.stdout.isTTY); // eslint-disable-line no-process-env
+        return (color === 'off') || (color !== 'on') && (!require('tty').isatty(1));
```

Comment:
What's the difference between `process.stdout.isTTY` and `require('tty').isatty(1)`? 🤔 

I feel this will have the same behaviour:
```suggestion
        return (color === 'off') || (color !== 'on') && (!process.stdout.isTTY);
```

Reply:
I searched some time ago and I found that isTTY not always works as good as isatty works across all the different node versions. I'll try to find it again, and let you know.

Reply:
Here it is:

https://github.com/babel/babel/issues/3763
https://github.com/nodejs/node/issues/28386